src/resources/postcss/tribe-common-admin/_main.pcss

```diff
@@ -434,6 +434,21 @@ table.plugins {
 	width: 68%;
 }
 
+.tribe-admin-image_field-btn-remove {
```

Comment:
I feel the naming here should be `.tribe-admin__settings-image-field-btn-remove` (if we don't go with `tec-` as the prefix)